demo/dynamic_embedding/amazon-us-reviews-digital-video-games/README.md

```diff
@@ -13,6 +13,6 @@ It will produce a model to `export_dir`.
 
```

Comment:
I think the demo name should contain "eager", make user understand easily.

Reply:
Agree